src/autoscaler/cf/app_test.go

```diff
@@ -17,6 +18,12 @@ import (
 	"net/url"
 )
 
+type CFSummaryResponse struct {
```

Comment:
it looks like the error response struct, then the name CFSummaryResponse is confusing